### ISSUE TITLE
[BUG]: DownloadPipelineArtifact@2 not working ("cannot access ... because it is being used by another process")

### ISSUE THREAD

**Issue by pyramid-scheme-ceo** [microsoft/azure-pipelines-tasks#20567]
### New issue checklist

- [x] I searched for [existing GitHub issues](https://github.com/microsoft/azure-pipelines-tasks/issues)
- [x] I read [pipeline troubleshooting guide](https://docs.microsoft.com/vsts/build-release/actions/troubleshooting)
- [x] I checked [how to collect logs](https://learn.microsoft.com/azure/devops/pipelines/troubleshooting/review-logs?view=azure-devops)

### Task name

DownloadPipelineArtifact@2

### Task version

2.198.0

### Issue Description

Our pipeline has started failing to Download artifacts today for only one of the three jobs we run. Both jobs run Ubuntu 20.04 and are running the same version of the task (2.198.0) yet one fails and the other succeeds.

The jobs are:
1. Build a JS component library and publish as an artifact
2. Download artifact and build "UI A"
3. Download artifact and build "UI B"

Jobs 2 and 3 run in parallel.

The download artifact step always works in job 2 and always fails in job 3 with an error similar to this:
```
##[error]The process cannot access the file '/home/vsts/work/1/s/frontend/node_modules/js-yaml/bin/js-yaml.js' because it is being used by another process.
```

Note that the error still happens when hitting re-running failed jobs (meaning that job 3 is running by itself and no longer in parallel with job 2).

The step is defined in yaml as

```
          - task: DownloadPipelineArtifact@2
            displayName: 'Download Artifact: web-library'
            inputs:
              buildType: current
              artifactName: WebLibrary
              targetPath: '$(Pipeline.Workspace)/s/frontend'
```

Idk if it's relevant but it's weird to me that the logs seems to say that it downloads the artifact three times (may just be automatic retries, or it could genuinely be trying to download three times at once leading to the exception).

### Environment type (Please select at least one enviroment where you face this issue)

- [ ] Self-Hosted
- [x] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

### Operation system

Ubuntu 20.04

### Relevant log output

2024-10-18T01:01:48.1228874Z ##[section]Starting: Download Artifact: web-library
2024-10-18T01:01:48.1239540Z ==============================================================================
2024-10-18T01:01:48.1239729Z Task         : Download Pipeline Artifacts
2024-10-18T01:01:48.1239833Z Description  : Download build and pipeline artifacts
2024-10-18T01:01:48.1239951Z Version      : 2.198.0
2024-10-18T01:01:48.1240059Z Author       : Microsoft Corporation
2024-10-18T01:01:48.1240162Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/download-pipeline-artifact
2024-10-18T01:01:48.1240347Z ==============================================================================
2024-10-18T01:01:48.5003431Z Download from the specified build: #29147
2024-10-18T01:01:48.5012319Z Download artifact to: /home/vsts/work/1/s/frontend
2024-10-18T01:01:49.0092512Z Using default max parallelism.
2024-10-18T01:01:49.3305795Z Using default max parallelism.
2024-10-18T01:01:49.3315173Z Max dedup parallelism: 192
2024-10-18T01:01:49.3315468Z DomainId: 0
2024-10-18T01:01:49.5066287Z ApplicationInsightsTelemetrySender will correlate events with X-TFS-Session b8bd3eea-be52-4afd-bf27-b689d1efef97
2024-10-18T01:01:49.5220111Z Hashtype: Dedup1024K
2024-10-18T01:01:49.5468277Z DedupManifestArtifactClient will correlate http requests with X-TFS-Session b8bd3eea-be52-4afd-bf27-b689d1efef97
2024-10-18T01:01:49.5486666Z Minimatch patterns: [**]
2024-10-18T01:01:52.7566041Z Filtered 212943 files from the Minimatch filters supplied.
2024-10-18T01:01:52.7937816Z Downloaded 0.0 MB out of 1,491.6 MB (0%).
2024-10-18T01:01:57.7209800Z DedupManifestArtifactClient will correlate http requests with X-TFS-Session b8bd3eea-be52-4afd-bf27-b689d1efef97
2024-10-18T01:01:57.7211262Z Minimatch patterns: [**]
2024-10-18T01:01:57.8023489Z Downloaded 18.2 MB out of 1,491.6 MB (1%).
2024-10-18T01:02:00.4086215Z Filtered 212943 files from the Minimatch filters supplied.
2024-10-18T01:02:00.4356045Z Downloaded 0.0 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:02.8219624Z Downloaded 0.4 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:05.3630500Z DedupManifestArtifactClient will correlate http requests with X-TFS-Session b8bd3eea-be52-4afd-bf27-b689d1efef97
2024-10-18T01:02:05.3631217Z Minimatch patterns: [**]
2024-10-18T01:02:05.4342391Z Downloaded 7.5 MB out of 1,491.6 MB (1%).
2024-10-18T01:02:07.8217646Z Downloaded 45.4 MB out of 1,491.6 MB (3%).
2024-10-18T01:02:08.8305651Z Filtered 212943 files from the Minimatch filters supplied.
2024-10-18T01:02:08.8600550Z Downloaded 0.0 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:10.4782365Z Downloaded 2.4 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:12.8315497Z Downloaded 17.7 MB out of 1,491.6 MB (1%).
2024-10-18T01:02:13.8563909Z Downloaded 17.7 MB out of 1,491.6 MB (1%).
2024-10-18T01:02:15.4806993Z Downloaded 17.7 MB out of 1,491.6 MB (1%).
2024-10-18T01:02:16.3305178Z DedupManifestArtifactClient will correlate http requests with X-TFS-Session b8bd3eea-be52-4afd-bf27-b689d1efef97
2024-10-18T01:02:16.3305905Z Minimatch patterns: [**]
2024-10-18T01:02:17.8401104Z Downloaded 55.6 MB out of 1,491.6 MB (4%).
2024-10-18T01:02:18.8531063Z Filtered 212943 files from the Minimatch filters supplied.
2024-10-18T01:02:18.8587161Z Downloaded 0.0 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:18.8789027Z Downloaded 0.0 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:20.4834981Z Downloaded 1.4 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:22.8395907Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:23.8592996Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:23.8800270Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:25.4828261Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:27.8352090Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:28.8554510Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:28.8757898Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:30.4855085Z Downloaded 4.1 MB out of 1,491.6 MB (0%).
2024-10-18T01:02:31.3926950Z ApplicationInsightsTelemetrySender correlated 2 events with X-TFS-Session b8bd3eea-be52-4afd-bf27-b689d1efef97
2024-10-18T01:02:31.3978588Z ##[error]The process cannot access the file '/home/vsts/work/1/s/frontend/node_modules/js-yaml/bin/js-yaml.js' because it is being used by another process.
2024-10-18T01:02:31.5266723Z ##[section]Finishing: Download Artifact: web-library

### Full task logs with system.debug enabled

(Fyi this log only shows the first instance of the exception otherwise it goes past the max limit of this field)

<details>
  <pre>
2024-10-18T01:22:49.3132053Z ##[debug]Evaluating condition for step: 'Download Artifact: web-library'
2024-10-18T01:22:49.3133081Z ##[debug]Evaluating: SucceededNode()
2024-10-18T01:22:49.3133404Z ##[debug]Evaluating SucceededNode:
2024-10-18T01:22:49.3133915Z ##[debug]=> True
2024-10-18T01:22:49.3134235Z ##[debug]Result: True
2024-10-18T01:22:49.3134576Z ##[section]Starting: Download Artifact: web-library
2024-10-18T01:22:49.3146062Z ==============================================================================
2024-10-18T01:22:49.3146258Z Task         : Download Pipeline Artifacts
2024-10-18T01:22:49.3146365Z Description  : Download build and pipeline artifacts
2024-10-18T01:22:49.3146674Z Version      : 2.198.0
2024-10-18T01:22:49.3146780Z Author       : Microsoft Corporation
2024-10-18T01:22:49.3146889Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/download-pipeline-artifact
2024-10-18T01:22:49.3147056Z ==============================================================================
2024-10-18T01:22:49.7063824Z ##[debug]TargetPath: /home/vsts/work/1/s/frontend
2024-10-18T01:22:49.7081861Z ##[debug]ArtifactName: WebLibrary
2024-10-18T01:22:49.7082907Z ##[debug]BuildResult: Succeeded
2024-10-18T01:22:49.7083763Z ##[debug]Run: CurrentRun
2024-10-18T01:22:49.7094663Z ##[debug]ProjectId: 6d659a19-d7bc-4e88-bb1f-943c39b20b18
2024-10-18T01:22:49.7188453Z Download from the specified build: #29148
2024-10-18T01:22:49.7190822Z ##[debug]Processed: ##vso[task.setvariable variable=BuildNumber;issecret=False;]29148
2024-10-18T01:22:49.7191233Z Download artifact to: /home/vsts/work/1/s/frontend
2024-10-18T01:22:50.2337010Z Using default max parallelism.
2024-10-18T01:22:50.2703872Z Verbose, ArtifactHttpClientFactory.CreateVssHttpClient: ClientSettingsHttpClient with BaseUri: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/, MaxRetries:5, SendTimeout:00:05:00
2024-10-18T01:22:50.3680479Z Using default max parallelism.
2024-10-18T01:22:50.3687152Z Max dedup parallelism: 192
2024-10-18T01:22:50.3687980Z DomainId: 0
2024-10-18T01:22:50.4023489Z Verbose, ArtifactHttpClientFactory.CreateVssHttpClient: DedupStoreHttpClient with BaseUri: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/, MaxRetries:5, SendTimeout:00:05:00
2024-10-18T01:22:50.5625475Z Information, ApplicationInsightsTelemetrySender will correlate events with X-TFS-Session 78f80715-0a56-4f78-8493-ed18ce2d61f5
2024-10-18T01:22:50.5626984Z Verbose, Started ITelemetrySender BlobStoreApplicationInsightsTelemetrySender.
2024-10-18T01:22:50.5771586Z Hashtype: Dedup1024K
2024-10-18T01:22:50.5897101Z Verbose, PipelineArtifact.DownloadMultipleArtifactsAsync starting
2024-10-18T01:22:50.6012658Z Information, DedupManifestArtifactClient will correlate http requests with X-TFS-Session 78f80715-0a56-4f78-8493-ed18ce2d61f5
2024-10-18T01:22:50.6029874Z Information, Minimatch patterns: [**]
2024-10-18T01:22:54.0157397Z Information, Filtered 212943 files from the Minimatch filters supplied.
2024-10-18T01:22:54.0497959Z Verbose, Could not initialize dataport: Dataport will not be used as it is only currently available on Windows.
2024-10-18T01:22:54.0532023Z Information, Downloaded 0.0 MB out of 1,491.6 MB (0%).
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.9690937Z ##[debug]Agent environment resources - Disk: / Available 14145.81 MB out of 74244.74 MB, Memory: Used 2753.00 MB out of 6932.00 MB, CPU: Usage 68.66%
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.5956822Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0544654, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.5958599Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1332007, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.5960223Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11054.4654 }
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.5964710Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11133.2007 }
2024-10-18T01:22:55.6005342Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8856821, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6007044Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10885.6821 }
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.6075341Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1117665, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6084203Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11111.7665 }
2024-10-18T01:22:55.6095026Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0841492, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6100122Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11084.1492 }
2024-10-18T01:22:55.6112363Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1238950, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6118931Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11123.895 }
2024-10-18T01:22:55.6125812Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1938249, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6133612Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11193.8249 }
2024-10-18T01:22:55.6145928Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0542348, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6150084Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11054.2348 }
2024-10-18T01:22:55.6171973Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0837692, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6173473Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11083.7692 }
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.6232439Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0253601, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.6287356Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11025.3601 }
2024-10-18T01:22:55.6307212Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1934265, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6311810Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11193.4265 }
2024-10-18T01:22:55.6317241Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8187427, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6321494Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10818.7427 }
2024-10-18T01:22:55.6329850Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.9713836, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.6336678Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10971.3836 }
2024-10-18T01:22:55.6342270Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1378768, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6343784Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11137.8768 }
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.6442122Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.9745732, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6444619Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10974.5732 }
2024-10-18T01:22:55.6484667Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0096371, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6486475Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11009.6371 }
2024-10-18T01:22:55.6488011Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1096621, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6489475Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11109.6621 }
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.6576843Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0188641, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6592713Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11018.8641 }
2024-10-18T01:22:55.6602082Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8382529, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6607343Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10838.2529 }
2024-10-18T01:22:55.6617272Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8411168, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6619231Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10841.1168 }
2024-10-18T01:22:55.6620542Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8211187, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6621975Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10821.1187 }
2024-10-18T01:22:55.6623355Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0993556, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6625174Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11099.3556 }
2024-10-18T01:22:55.6626685Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0621947, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6628061Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11062.1947 }
2024-10-18T01:22:55.6629433Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0336772, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6630725Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11033.6772 }
2024-10-18T01:22:55.6645345Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.9428543, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6646846Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10942.8543 }
2024-10-18T01:22:55.6770362Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.0735757, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6771844Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11073.5757 }
2024-10-18T01:22:55.6800545Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:11.1600938, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6802040Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":11160.0938 }
2024-10-18T01:22:55.6858533Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8589122, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.6870744Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10858.9122 }
2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:54.8188138Z Verbose, Downloaded2024-10-18T01:22:55.7439342Z Verbose, ArtifactHttpRetryMessageHandler.TraceHttpRequestRetrying: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, attempt 1, backoff 00:00:10.8774619, HttpStatusCode.ServiceUnavailable, WebExceptionStatus., SocketError., WinHttpErrorCode., CurlErrorCode., activity 
2024-10-18T01:22:55.7441011Z Verbose, ArtifactHttpRetryMessageHandler.TraceRaw: https://vsblobprodeau1.vsblob.visualstudio.com/A4c207bb4-8f8a-4b1f-80fc-8690070ec048/_apis/dedup/urls?allowEdge=true, tracepoint 100011, TraceLevel.Error, message { "Client":"", "Endpoint":"vsblobprodeau1.vsblob.visualstudio.com", "Attempt":2, "MaxAttempts":6, "Backoff":10877.4619 }
</pre>
</details>

### Repro steps

stages:
  - stage: Build
    jobs:
      - job: Build_Web_Library
        pool:
          vmImage: ubuntu-20.04
        workspace:
          clean: all
        steps:
          - checkout: self
            fetchDepth: 1
            fetchTags: false
          - <NPM install/build/compile a Typescript/React project>
          - task: PublishPipelineArtifact@1
            displayName: 'Publish Artifact: web-library'
            inputs:
              targetPath: frontend
              artifact: WebLibrary
              publishLocation: pipeline
      
      - job: Build_App_A
        dependsOn: Build_Web_Library
        pool:
          vmImage: ubuntu-20.04
        workspace:
          clean: all
        steps:
          - checkout: self
            fetchDepth: 1
            fetchTags: false

          # This works
          - task: DownloadPipelineArtifact@2
            displayName: 'Download Artifact: web-library'
            inputs:
              buildType: current
              artifactName: WebLibrary
              targetPath: '$(Pipeline.Workspace)/s/frontend'

      - job: Build_App_B
        dependsOn: Build_Web_Library
        pool:
          vmImage: ubuntu-20.04
        workspace:
          clean: all
        steps:
          - checkout: self
            fetchDepth: 1
            fetchTags: false

          # This fails
          - task: DownloadPipelineArtifact@2
            displayName: 'Download Artifact: web-library'
            inputs:
              buildType: current
              artifactName: WebLibrary
              targetPath: '$(Pipeline.Workspace)/s/frontend'

**Comment by DeadlyDragon** [comment]
I started running into this today as well but it's on a self hosted agent. Last successful run was yesterday, few runs today mostly on the same pipelines, others are working

**Comment by sebastienchoisy** [comment]
Hello, started running into the same issue since yesterday as well

**Comment by ashagraw91** [comment]
we are also facing same issue , what is quick fix?

**Comment by MTomBosch** [comment]
Also in my company we are facing this issue now more and more. What helps to a certain degree is to use the RetryOnErrorCount feature on those steps.

**Comment by sebastienchoisy** [comment]
Hello guys,
The only workaround I found for now is to download my artifact with the download step.
 - https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-download?view=azure-pipelines
Looks like that one is working (was able to download 388mb).

As you can't set a target path wit the download step, I added an extra step after to copy (CopyFiles@2) my downloaded artifact to the target path I was using with DownloadPipelineArtifact@2.
Hope it helps !

**Comment by n-losse** [comment]
@MTomBosch are you on self hosted agents? According to the docs, the workspace on MS hosted ones is always clean https://learn.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#workspace

**Comment by MTomBosch** [comment]
Yes, we are on self-hosted agents.

**Comment by pyramid-scheme-ceo** [comment]
We've taken the trade-off of adding ~10 mins to our build for now by building the same artifact in each of the steps instead of building once and downloading the artifact in the two next steps. Would be great if there was any progress on this!

**Comment by Turbofroggy** [comment]
We are hitting this same issue with on-premises self-hosted Mac agents.

**Comment by viralpandya** [comment]
I am sorry for the inconvenience. Can we get agent version and also if possible logs for failed pipeline as well?

**Comment by pyramid-scheme-ceo** [comment]
@viralpandya there are agent versions (ubuntu 20.04) and both simple and verbose logs in the original post. Is there anything else you require that I can provide?

**Comment by ANURAGKUNCHAMWAR** [comment]
There was some ghost error about a particular file which was being complained of unable to download or still being used, and so after trying alot of things, deleted and recreated the file in a new branch and new release tag and then the issue got resolved

**Comment by ANURAGKUNCHAMWAR** [comment]
There was some ghost error about a particular file which was being complained of unable to download or still being used, and so after trying alot of things, deleted and recreated the file in a new branch and new release tag and then the issue got resolved

**Comment by viralpandya** [comment]
@james-woodbridge  - Once again sorry for inconvenience. But this issue is fixed, and it should be deployed soon and its not in 4.252.0 agent version
